tests/console/supportutils.pm

```diff
@@ -23,8 +23,9 @@ sub run {
     my $self = shift;
     $self->select_serial_terminal;
 
+    my $options = get_var('SUPPORTCOFIG_OPTIONS', '');
     assert_script_run "rm -rf nts_* scc_* ||:";
-    assert_script_run "supportconfig -t . -B test", 800;
+    assert_script_run "supportconfig $options -t . -B test", 2000;
```

Comment:
After using your desired options, are going to need 2000 seconds timeout still?

Reply:
No but timeout is not problem, it's still better to have higher timeout than have it low.
If command is killed due to openQA timeout then you don't know if the command would pass or fail ...